src/components/ActivityIndicator/index.tsx

```diff
@@ -37,17 +36,19 @@ const HiddenDiv = styled.div`
   position: absolute;
 `
 
+type Accent = keyof Theme['colors']
```

Comment:
```suggestion
type Accent = keyof Theme.colors
```

Reply:
Also I think it will be best to keep naming convention following design tokens, and so this variable should be called `Sentiment` more than `Accent` :)

Reply:
done, `Sentiment` type definition moved to `src/theme/colors/index.ts`